apps/mobile/src/state/README.md

```diff
@@ -0,0 +1,9 @@
+# App state (client-side state)
+
+We keep the app state using `zustand`.
+
+We have 3 types of store:
```

Comment:
Upon reflection, I think it could be a good idea to always protect all data in the store and require face ID to open the app, even if only to give users a more secure feeling. We could then also ask for it again when performing key interactions. 

Maybe that would simplify the implementation and we could just have a single protected store for all data? At least for persisted and protected. I'm not sure if temporary data even needs to be in a store, could that be inside application state instead?

I see you mention it's easier to have different middleware for different stores than slices so maybe that's not possible?

Do we have a fallback option where faceID is not supported? e.g. fingerprint / pin?